.gitlab/build.yml

```diff
@@ -103,124 +57,98 @@ calculate-build-metadata:
     TMPL_SRC_REPO: ${ADP_IMAGE_REPO_NAME}
-    TMPL_SRC_IMAGE: ${CI_COMMIT_SHORT_SHA}
+    TMPL_SRC_IMAGE: "v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}"
     RELEASE_TAG: "v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}"
-    BUILD_TAG: "v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}"
+    BUILD_TAG: "v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-build"
     RELEASE_STAGING: "true"
     RELEASE_PROD: "true"
 
-upload-adp-symbols:
-  extends: [.build-common-variables, .upload-adp-symbols-definition]
-  needs:
-    - build-adp-image
-  variables:
-    IMAGE_TAG: ${ADP_DEBUG_IMAGE_TAG}
-
-upload-adp-symbols-fips:
-  extends: [.build-common-variables, .upload-adp-symbols-definition]
-  needs:
-    - build-adp-image-fips
-  variables:
-    IMAGE_TAG: ${ADP_DEBUG_IMAGE_TAG_FIPS}
-
-upload-adp-release-symbols:
-  extends: [.build-common-variables, .upload-adp-symbols-definition]
-  rules:
-    - if: !reference [.on_official_release, rules, if]
-  needs:
-    - build-adp-image-release
-  variables:
-    IMAGE_TAG: ${ADP_RELEASE_IMAGE_TAG}
-
-upload-adp-release-symbols-fips:
-  extends: [.build-common-variables, .upload-adp-symbols-definition]
-  rules:
-    - if: !reference [.on_official_release, rules, if]
+publish-adp-image-internal-fips:
+  stage: build
+  extends: [.build-common-variables]
   needs:
-    - build-adp-image-release-fips
+    - build-adp-image-internal-fips
+  trigger:
+    project: DataDog/images
+    branch: master
+    strategy: depend
   variables:
-    IMAGE_TAG: ${ADP_RELEASE_IMAGE_TAG_FIPS}
+    IMAGE_NAME: agent-data-plane
+    IMAGE_VERSION: tmpl-v1
+    TMPL_SRC_REPO: ${ADP_IMAGE_REPO_NAME}
+    TMPL_SRC_IMAGE: "v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-fips"
+    RELEASE_TAG: "v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-fips"
+    BUILD_TAG: "v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-fips-build"
+    RELEASE_STAGING: "true"
+    RELEASE_PROD: "true"
 
 display-image-tags:
   extends: [.build-common-variables]
   stage: build
   needs:
-    - build-adp-image
-    - build-adp-image-fips
-    - build-converged-adp-image
-    - build-converged-adp-image-fips
-    - publish-internal-adp-image
+    - build-adp-image-internal
+    - build-adp-image-internal-fips
+    - publish-adp-image-internal
+    - publish-adp-image-internal-fips
   script:
     - |-
       cat <<EOF
-      # Image Tags
-
-      ## Barebones (ADP only, not suitable for deployments)
-      Release (w/ debuginfo, non-FIPS): ${ADP_DEBUG_IMAGE_TAG}
-      Release (w/ debuginfo, FIPS):     ${ADP_DEBUG_IMAGE_TAG_FIPS}
+      # ADP Image Tags
 
-      ## Internal (ADP only, baked with necessary tools for internal deployments)
-      Release (w/ debuginfo, non-FIPS): ${IMAGE_REGISTRY}/agent-data-plane:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}
+      ## Internal (baked with necessary tools for internal deployments)
+      Non-FIPS: ${IMAGE_REGISTRY}/agent-data-plane:${ADP_INTERNAL_IMAGE_TAG}
+      FIPS:     ${IMAGE_REGISTRY}/agent-data-plane:${ADP_INTERNAL_IMAGE_TAG_FIPS}
```

Comment:
For my curiosity, what do these lines do?

Reply:
We're just outputting the image tags that have been built so it's easier to find them in the CI pipeline when you then want to go and deploy the built image internally.

Reply:
Ohh is just print statements 😄